### DISCUSSION TITLE
Letters do not reach the recipient

### DISCUSSION THREAD

**Discussion by kargo123456** [tutao/tutanota#8157] [Bugs] [answered]
Recently, letters have stopped reaching many recipients. Although previously everything reached these email addresses

As a result, this message arrives

"This is the mail system at host mail.w14.tutanota.de.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

The mail system"

**Comment by charlag** [comment, marked as answer]
Hi please contact support if you are a premium customer or hello at tutao de if you aren't

**Reply by kargo123456** [reply]
Hi! I don't quite understand where to go with this question if I'm not a premium client?

**Reply by charlag** [reply]
hello [at] tutao de

**Reply by kargo123456** [reply]
Is this support service? Right?

**Reply by hrb-hub** [reply]
> Is this support service? Right?

Yes
